test/models/piece_test.rb

```diff
@@ -80,8 +80,15 @@ def setup
     @pawn_black = Piece.where(row_position: 6, col_position: 0).first
     @pawn_black.update(captured: true)
```

Comment:
Since the method no longer checks for if the piece is captured, this doesn't make sense any more.